llvm/include/llvm/CodeGen/Analysis.h

```diff
@@ -148,10 +148,6 @@ bool returnTypeIsEligibleForTailCall(const Function *F, const Instruction *I,
                                      const TargetLoweringBase &TLI,
                                      bool ReturnsFirstArg = false);
 
-/// Check whether B is a bitcast of a pointer type to another pointer type,
-/// which is equal to A.
-bool isPointerBitcastEqualTo(const Value *A, const Value *B);
```

Comment:
Removing this is an unrelated patch?

Reply:
Eli commented in another part of the patch that given we have opaque pointers now this isn't needed anymore.